CMakeLists.txt

```diff
@@ -21,7 +21,12 @@ if (WIN32)
   option(BUILD_PYTHON_BINDINGS "Build Python bindings." OFF)
   option(BUILD_SHARED_LIBS
       "Compile shared libraries (if OFF, static libraries are compiled)." OFF)
-  message(WARNING "By default Python bindings are not compiled for Windows because they are not known to work.  Set BUILD_PYTHON_BINDINGS to ON if you want them built.")
+  if (NOT BUILD_PYTHON_BINDINGS)
+    message(WARNING "By default Python bindings are not compiled for Windows because they are not known to work.  Set BUILD_PYTHON_BINDINGS to ON if you want them built.")
```

Comment:
I think this warning should be deprecated right?

Reply:
Good point, thanks!  Let me fix that.